test/android/android.fixtures.ts

```diff
@@ -32,4 +34,11 @@ fixtures.device.init(async ({ playwright }, runTest) => {
   await device.close();
 });
 
+fixtures.page.override(async ({ device }, runTest) => {
+  await device.shell('am start org.chromium.webview_shell/.WebViewBrowserActivity');
```

Comment:
That's a good idea, but we would also want to test browser. Should we run this app only once? We can later add our own testing apk that gives new webview for each test...

Reply:
yup reverted for now. Lets focus in this PR only that we can run existing tests no Android.